src/core/tritonserver.cc

```diff
@@ -289,7 +289,7 @@ class TritonServerOptions {
 };
 
 TritonServerOptions::TritonServerOptions()
-    : server_id_("inference:0"),
+    : server_id_("triton"),
```

Comment:
Could you also propagate these changes to L0_lifecycle, L0_server_status, server.cc and simple_grpc/http_health_metadata.cc/py?

Reply:
Yes, I will fix those/